### PR TITLE
Return all attributes when querying for a graph's nodes

### PR DESCRIPTION
**Pull request by waxlamp**
TODO:
- [x] Either add field filtering option, or open an issue around doing the same
- [x] Release multinetjs
- [x] Update multinetjs version number in client

### CODE REVIEW

File: multinetjs/src/index.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import { Client } from './client';
 
+export interface TableRow {
+  _key: string;
+  _id: string;
+}
+
```
Comment (JackWilb) [inline multinetjs/src/index.ts]: Would this ever break? Can the TableRow have more than just _key and  _id? Will it have attributes?

Comment (waxlamp) [reply multinetjs/src/index.ts]: The beauty of TypeScript "interfaces" is they're composable. That is, saying that a value is a `TableRow` means that it must have `_key` and `_id` keys, but can also have other things in it as well. If you had a specific type that just encoded someone's name and age, you could do:

```typescript
interface Person {
  name: string;
  age: number;
}
```

and then declare a specific table's rows as having type `TableRow & Person`. But in general, a `TableRow` can indeed have other attributes on it.

Comment (JackWilb) [reply multinetjs/src/index.ts]: Awesome, thanks for the clarification

**Comment by waxlamp** [comment]
Hey guys, after the multinetjs updates, I need another review here. Sorry about that.

**Pull request merged**